projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/base/integers/LLVMI32Node.java

```diff
@@ -36,6 +36,7 @@
 
     public static final long MASK = 0xffffffffL;
     public static final int BYTE_SIZE = 4;
+    public static final int BIT_SIZE = 32;
```

Comment:
Yeah, if you do a rebase, you will already find this field renamed (and moved to LLVMExpressionNode).